tests/scripts/all.sh

```diff
@@ -1335,6 +1341,173 @@ component_build_psa_want_ecdsa_disabled_software() {
     make CC=gcc CFLAGS="$ASAN_CFLAGS -DPSA_CRYPTO_DRIVER_TEST -DMBEDTLS_PSA_ACCEL_ALG_ECDSA -DMBEDTLS_PSA_ACCEL_ALG_DETERMINISTIC_ECDSA -I../tests/include -O2" LDFLAGS="$ASAN_CFLAGS"
 }
 
+# This should be renamed to test and updated once the accelerator ECDH code is in place and ready to test.
```

Comment:
I haven't reviewed the `all.sh` components in detail. This long list doesn't scale: we need a script like the existing `depends-xxx.pl` family. It's ok to delay this script until after this PR, and in fact I think we should delay merging such a script until we have systematic testing for the driver interface as Ronald started to do in https://github.com/ARMmbed/mbedtls/pull/3744. In the meantime, I question the usefulness of this long list of ad hoc configurations that aren't actually in their final form.

Reply:
@gilles-peskine-arm Would it be best to simply remove the testing for now then?

Reply:
Well, we do want to validate builds. I don't have a better short-term solution, so please go ahead with these ad hoc builds for now.